docs/src/arithmetic.md

```diff
@@ -1,9 +1,11 @@
+# Arithmetic
+
 ## ZonedDateTime-Period Arithmetic
 
-`ZonedDateTime` uses calendrical arithmetic in a [similar manner to `DateTime`](https://docs.julialang.org/en/stable/manual/dates/#TimeType-Period-Arithmetic-1) but with some key differences. Lets look at these differences by adding a day to March 30th 2014 in Europe/Warsaw.
+`ZonedDateTime` uses calendrical arithmetic in a [similar manner to `DateTime`](https://docs.julialang.org/en/stable/stdlib/Dates/#TimeType-Period-Arithmetic-1) but with some key differences. Lets look at these differences by adding a day to March 30th 2014 in Europe/Warsaw.
 
-```julia
-julia> using Base.Dates
+```jldoctest spring
+julia> using TimeZones, Compat.Dates
```

Comment:
Can we just have the doctests run against Julia 1.0?

Reply:
I think I'd prefer: `using TimeZones, Dates`

Reply:
Sure